### ISSUE TITLE
Spicetify is now only showing the body element

### ISSUE THREAD

**Issue by HeyImKoa1a** [spicetify/spicetify-cli#1593]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: 1.1.81.604.gccacfc8c
- Spicetify version: 2.9.5
```

### 📝 Description

I was changing around my theme, which I haven't done in a little while, however, for some reason as soon as I tried to changed the Theme, spicetify completely stopped loading everything but the body element, I went back to my old Theme that I know works, but the same issue persisted

I have tried turning off settings in the config such as overriding elements and injecting CSS, but the same issue always applies, I am able to "spicetify restore backup" and then spotify is back, but without CSS or extensions of course, it seems to be an issue with spicetify itself

Thanks in advance :)

### 📸 Screenshots

![Screenshot_1](https://user-images.githubusercontent.com/75582908/164078651-b7f9393d-ea95-4d94-b711-b9dc2a92d0b5.png)
![Screenshot_2](https://user-images.githubusercontent.com/75582908/164078655-b02bd315-a4cd-466c-b9ae-580edc651aad.png)

**Comment by ririxidev** [comment]
Please check errors in DevTools console. How to get DevTools running in latest spotify version can be found here: https://github.com/spicetify/spicetify-cli/issues/1518#issuecomment-1060122063

**Comment by HeyImKoa1a** [comment]
![image](https://user-images.githubusercontent.com/75582908/164082692-8d1a7a9c-27c4-4efb-b432-a670aefc61cb.png)

These are the only thrown errors, and it will not let me restart the page either

**Comment by ririxidev** [comment]
I recommend reinstalling spotify fully with removing spotify folders :D

**Comment by HeyImKoa1a** [comment]
I have already tried that, and I just went through it again, but it seems to still be happening, still throwing the same error, I tried to look through XPUI.js, but since its obfuscated I doubt I'll be able to get anywhere with that

![image](https://user-images.githubusercontent.com/75582908/164084532-bd98ee28-7940-413c-82de-f7ccf38daf6a.png)

**Comment by HeyImKoa1a** [comment]
![image](https://user-images.githubusercontent.com/75582908/164085297-f2f4b366-0456-4caf-9a4c-82dd648a931c.png)

Just tried reinstalling spicetify as well, deleted all files and made a fresh install, but it still has the issue, even with no extensions, apps, themes or settings being changed

**Comment by gibmadikka** [comment]
Experiencing the same issue, tried backups and clean install.

**Comment by AutumnRivers** [comment]
Can confirm I'm having this issue as well.

Spotify for Windows
1.1.83.956.gbe7ea9a7

spicetify -v
2.9.5

**Comment by ririxidev** [comment]
1.1.83 isn't supported by spicetify for now.

**Comment by ririxidev** [comment]
> I have already tried that, and I just went through it again, but it seems to still be happening, still throwing the same error, I tried to look through XPUI.js, but since its obfuscated I doubt I'll be able to get anywhere with that
> 
> ![image](https://user-images.githubusercontent.com/75582908/164084532-bd98ee28-7940-413c-82de-f7ccf38daf6a.png)

That's actually, weird. 1.1.81 is supported by spicetify so that shouldn't be problem. So without anything it still doesn't load huh

**Comment by xerta555** [comment]
Ok so no `1.1.82` version, directly `1.1.83` and need a new patch.
Can't wait to see if this new version solves the bugs of the previous version.

**Comment by HeyImKoa1a** [comment]
> > I have already tried that, and I just went through it again, but it seems to still be happening, still throwing the same error, I tried to look through XPUI.js, but since its obfuscated I doubt I'll be able to get anywhere with that
> > ![image](https://user-images.githubusercontent.com/75582908/164084532-bd98ee28-7940-413c-82de-f7ccf38daf6a.png)
> 
> That's actually, weird. 1.1.81 is supported by spicetify so that shouldn't be problem. So without anything it still doesn't load huh and it only happens with spicetify?

yeah, only with spicetify, my normal spotify is fine, literally no issues, just spicetify
![image](https://user-images.githubusercontent.com/75582908/164319679-e242a199-e4df-4903-9869-668f1ba4b7a4.png)

**Comment by xerta555** [comment]
> > > I have already tried that, and I just went through it again, but it seems to still be happening, still throwing the same error, I tried to look through XPUI.js, but since its obfuscated I doubt I'll be able to get anywhere with that
> > > ![image](https://user-images.githubusercontent.com/75582908/164084532-bd98ee28-7940-413c-82de-f7ccf38daf6a.png)
> > 
> > 
> > That's actually, weird. 1.1.81 is supported by spicetify so that shouldn't be problem. So without anything it still doesn't load huh and it only happens with spicetify?
> 
> yeah, only with spicetify, my normal spotify is fine, literally no issues, just spicetify ![image](https://user-images.githubusercontent.com/75582908/164319679-e242a199-e4df-4903-9869-668f1ba4b7a4.png)

Experimental features work again?
For Spicetify compatibility, it need a patch.

**Comment by iOddishGX** [comment]
Same issue

**Comment by CharlieS1103** [comment]
1.82 and 1.83 are fine for me unfortunately. Sometimes the xpui.js simply does not update so I'll try to break it for myself(and then fix it).

**Comment by LweoIsHere** [comment]
same issue as well :c

**Comment by fernamp** [comment]
Same here.
I run "spicetify upgrade" after Spotify itself got an update: spicetify shows nothing.

**Versions**
Spicetify 2.95
Spotify 1.1.83.956

I had a previous Spotify setup file and used it with Spicetify 2.9.5. Everything is back to normal. Spotify 1.1.81.604 is working.

**Comment by xerta555** [comment]
> Same here. I run "spicetify upgrade" after Spotify itself got an update: spicetify shows nothing.
> 
> **Versions** Spicetify 2.95 Spotify 1.1.83.956
> 
> I had a previous Spotify setup file and used it with Spicetify 2.9.5. Everything is back to normal. Spotify 1.1.81.604 is working.

Did you do the test of make a complete fresh install of Spotify 1.1.83.956 and Spicetify before reverting back to previous version?

**Comment by fernamp** [comment]
> > Same here. I run "spicetify upgrade" after Spotify itself got an update: spicetify shows nothing.
> > **Versions** Spicetify 2.95 Spotify 1.1.83.956
> > I had a previous Spotify setup file and used it with Spicetify 2.9.5. Everything is back to normal. Spotify 1.1.81.604 is working.
> 
> Did you do the test of make a complete fresh install of Spotify 1.1.83.956 and Spicetify before reverting back to previous version?

I tried that, worked for one  view, closed spotify, opened again and then went into blackness.
I installed spotify 604 without removing anything, then run spicetify backup and spicetify apply. All good. I'll keep myself to this version and awaiting for the update for 956/295

**Comment by xerta555** [comment]
> > > Same here. I run "spicetify upgrade" after Spotify itself got an update: spicetify shows nothing.
> > > **Versions** Spicetify 2.95 Spotify 1.1.83.956
> > > I had a previous Spotify setup file and used it with Spicetify 2.9.5. Everything is back to normal. Spotify 1.1.81.604 is working.
> > 
> > 
> > Did you do the test of make a complete fresh install of Spotify 1.1.83.956 and Spicetify before reverting back to previous version?
> 
> I tried that, worked for one view, closed spotify, opened again and then went into blackness. I installed spotify 604 without removing anything, then run spicetify backup and spicetify apply. All good. I'll keep myself to this version and awaiting for the update for 956/295

604 ?
956/295 ?

It's now sure that Spietify need a patch to work with this new version.

**Comment by fernamp** [comment]
> It's now sure that Spietify need a patch to work with this new version.

I forgot to clarify but the new version of Spotify came out this the weekend (956) (more or less). So I was aware that something could go wrong. I have not tried another version but I´m 100% positive **Spotify 604 works with Spicetify 295**.

**Comment by ririxidev** [comment]
Spotify works fine for me with new update.
![image](https://user-images.githubusercontent.com/9348108/164459547-2f2a22b6-93b6-40b0-a680-c9190c01060a.png)


Environment: 
```
- spotify 1.1.83.956.gbe7ea9a7
- Windows 11 Pro Insider Preview (OS Build 22598.200)
- spicetify v2.9.5
```

**Comment by fernamp** [comment]
> Spotify + spicetify works fine for me with new update. ![image](https://user-images.githubusercontent.com/9348108/164459547-2f2a22b6-93b6-40b0-a680-c9190c01060a.png)
> 
> Environment:
> 
> ```
> - spotify 1.1.83.956.gbe7ea9a7
> - Windows 11 Pro Insider Preview (OS Build 22598.200)
> - spicetify v2.9.5
> ```

Do you see marketplace? Or don't you use it?

**Comment by xerta555** [comment]
> Spotify + spicetify works fine for me with new update. ![image](https://user-images.githubusercontent.com/9348108/164459547-2f2a22b6-93b6-40b0-a680-c9190c01060a.png)
> 
> Environment:
> 
> ```
> - spotify 1.1.83.956.gbe7ea9a7
> - Windows 11 Pro Insider Preview (OS Build 22598.200)
> - spicetify v2.9.5
> ```

You don't get contents (right panel of Spotify)

**Comment by ririxidev** [comment]
> > Spotify + spicetify works fine for me with new update. ![image](https://user-images.githubusercontent.com/9348108/164459547-2f2a22b6-93b6-40b0-a680-c9190c01060a.png)
> > Environment:
> > ```
> > - spotify 1.1.83.956.gbe7ea9a7
> > - Windows 11 Pro Insider Preview (OS Build 22598.200)
> > - spicetify v2.9.5
> > ```
> 
> You don't get contents (right panel of Spotify)

I disabled it 🤦‍♀️

**Comment by ririxidev** [comment]
> > Spotify + spicetify works fine for me with new update. ![image](https://user-images.githubusercontent.com/9348108/164459547-2f2a22b6-93b6-40b0-a680-c9190c01060a.png)
> > Environment:
> > ```
> > - spotify 1.1.83.956.gbe7ea9a7
> > - Windows 11 Pro Insider Preview (OS Build 22598.200)
> > - spicetify v2.9.5
> > ```
> 
> Do you see marketplace? Or don't you use it? I'll test to remove all and install it again. BUT, I will not take it as a solution, I mean, everytime I do this I have to re-config the extensions. Spicetify easy install is not a solution for me, cause the company antivirus removes the EXE.

![image](https://user-images.githubusercontent.com/9348108/164463512-2fe8bbfb-595e-4e97-8eba-a9a734e81073.png)
works just fine

**Comment by xerta555** [comment]
> > > Spotify + spicetify works fine for me with new update. ![image](https://user-images.githubusercontent.com/9348108/164459547-2f2a22b6-93b6-40b0-a680-c9190c01060a.png)
> > > Environment:
> > > ```
> > > - spotify 1.1.83.956.gbe7ea9a7
> > > - Windows 11 Pro Insider Preview (OS Build 22598.200)
> > > - spicetify v2.9.5
> > > ```
> > 
> > 
> > Do you see marketplace? Or don't you use it? I'll test to remove all and install it again. BUT, I will not take it as a solution, I mean, everytime I do this I have to re-config the extensions. Spicetify easy install is not a solution for me, cause the company antivirus removes the EXE.
> 
> ![image](https://user-images.githubusercontent.com/9348108/164463512-2fe8bbfb-595e-4e97-8eba-a9a734e81073.png) works just fine

Themes, experimental features panel and DevTools works again with this version?
I don't remember me that there is a feature to hidden contents.

**Comment by ririxidev** [comment]
DevTools will not work, ever again. It needs to be patched in memory if you even want to use it.
Experimental features are still broken. No one said that it works. It needs to be fixed.

**Comment by xerta555** [comment]
> DevTools will not work, ever again. It needs to be patched in memory if you even want to use it. Experimental features are still broken. No one said that it works. It needs to be fixed. ![image](https://user-images.githubusercontent.com/9348108/164464222-eba2a4c3-a888-4f2f-b6ed-137213aaec99.png) to disable friends panel... -_-

Why ever again? It's CEF part, no?
For other features i didn't say that they works again, so i asked.

**Comment by ririxidev** [comment]
> Why ever again? It's CEF part, no?
It needs to be patched in memory if you even want to use it. 
or patched binary but that probably will not work :D

**Comment by tonytkachenko** [comment]
Fully reinstalled Spotify and spicetify, removed config folders, rebooted the system, and... Nothing :D Got the [same blank screen](https://github.com/spicetify/spicetify-marketplace/issues/169)

Windows 11
Spicetify: 2.9.5
Spotify: 1.1.83.956.gbe7ea9a7

**Comment by tracedbyt3** [comment]
> Fully reinstalled Spotify and spicetify, removed config folders, rebooted the system, and... Nothing :D Got the [same blank screen](https://github.com/spicetify/spicetify-marketplace/issues/169)
> 
> Windows 11 Spicetify: 2.9.5 Spotify: 1.1.83.956.gbe7ea9a7

Exact same thing here. Did everything mentioned and still blank. Spotify Version: 1.1.83.956.gbe7ea9a7, Spicetify Version: 2.9.5, windows 10

**Comment by ririxidev** [comment]
I think it has been resolved by #1600 in spicetify v2.9.6. Please upgrade and let us know!

**Comment by ririxidev** [comment]
Closing since issue has been resolved.
